scripts/stablex/place_spread_orders.js

```diff
@@ -0,0 +1,128 @@
+const BatchExchange = artifacts.require("BatchExchange")
+const fetch = require("node-fetch")
+const BN = require("bn.js")
+const readline = require("readline")
+
+const { sendTxAndGetReturnValue } = require("../../test/utilities.js")
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+})
+
+const token_list_url = "https://raw.githubusercontent.com/gnosis/dex-js/master/src/tokenList.json"
```

Comment:
this list already exists somewhere in the repo

Reply:
The list doesn't, but this link does.

Reply:
The list itself not, but the URL yes. It is here.

https://github.com/gnosis/dex-contracts/blob/df44013a89206a893ee616203f1833960baff0bc/scripts/stablex/add_token_list.js#L3-L7

Should this become a common environment var?

Reply:
Just a shared constant in `utilities.js` would be fine. Let's just make sure we won't accidentally update the list in one place and not the other.